src/steps/os/linux.rs

```diff
@@ -74,6 +76,11 @@ impl Distribution {
             Some("openmandriva") => Distribution::OpenMandriva,
             Some("pclinuxos") => Distribution::PCLinuxOS,
             _ => {
+                if let Some(name) = name {
```

Comment:
They don't have a unique `ID` for now, so we have to check the `NAME` field, but they will have one when they reach version 2.0, and we can preview it [here](https://github.com/Vanilla-OS/base-files/blob/2414c26f325e78295ee6e59a89b199fdd7d8f850/etc/os-release#L6).